src/compiler/commandLineParser.ts

```diff
@@ -141,6 +141,14 @@ namespace ts {
             category: Diagnostics.Advanced_Options,
             description: Diagnostics.Show_verbose_diagnostic_information
         },
+        {
+            name: "generateCPUProfile",
+            type: "string",
+            isFilePath: true,
+            paramType: Diagnostics.FILE,
+            category: Diagnostics.Advanced_Options,
+            description: Diagnostics.Show_verbose_diagnostic_information

```

Comment:
Copy-paste error?

Reply:
Ah, yeah, meant to add a new one for that~